[caffe2] Use AVX2 to increase frequency for FP16<->FP32 Caffe2 ops

jianyuh opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31203 [caffe2] Use AVX2 to increase frequency for FP16<->FP32 Caffe2 ops**

For multi-instance environment, AVX2 should help increase the clock frequency.

Differential Revision: [D18962649](https://our.internmc.facebook.com/intern/diff/D18962649/)

jianyuh committed:
[caffe2] Use AVX2 to increase frequency for FP16<->FP32 Caffe2 ops

For multi-instance environment, AVX2 should help increase the clock frequency.

Differential Revision: [D18962649](https://our.internmc.facebook.com/intern/diff/D18962649/)

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit 49bff18:
* **1/1** broken upstream at merge base 2488231 (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=2488231fe32e6e2c86f58acaa34766a6daa19235&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
    * You may want to rebase on the `viable/strict` branch (see its [recency history](https://dr.pytorch.org/master-viable-commits.html)):
        * If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
          ```
          git fetch viable/strict
          git rebase --onto viable/strict $(git merge-base origin/master HEAD)
          ```
        * If your commit is older than `viable/strict`:
          ```
          git fetch viable/strict
          git rebase viable/strict
          ```
* **0/1** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=49bff18f2ff53303e6f401ed1317081d35034894).

### 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably caused by upstream breakages:
* [pytorch_xla_linux_xenial_py3_6_clang7_build](https://dr.pytorch.org/build-details.html?build_id=73398564)



---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

jianyuh committed:
Update on "[caffe2] Use AVX2 to increase frequency for FP16<->FP32 Caffe2 ops"


For multi-instance environment, AVX2 should help increase the clock frequency.

Differential Revision: [D18962649](https://our.internmc.facebook.com/intern/diff/D18962649/)

[ghstack-poisoned]

jianyuh committed:
Update on "[caffe2] Use AVX2 to increase frequency for FP16<->FP32 Caffe2 ops"


For multi-instance environment, AVX2 should help increase the clock frequency.

Differential Revision: [D18962649](https://our.internmc.facebook.com/intern/diff/D18962649/)

[ghstack-poisoned]

jianyuh committed:
Update on "[caffe2] Use AVX2 to increase frequency for FP16<->FP32 Caffe2 ops"


For multi-instance environment, AVX2 should help increase the clock frequency.

Differential Revision: [D18962649](https://our.internmc.facebook.com/intern/diff/D18962649/)

[ghstack-poisoned]
